src/Microsoft.DotNet.InternalAbstractions/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for making code that uses file system and environment testable.",
-  "version": "1.0.0-preview2-1-*",
+  "version": "1.0.500",
```

Comment:
I would still put pre-release versions on these, since:

1. We are going to build multiple builds before actually shipping.
2. The library depends on pre-release versions of corefx libraries - so it can't be a release build.  (There's a rule in NuGet that a released package can't depend on other pre-release packages.)